src/mapping/AxialGeoMultiscaleMapping.hpp

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include "mapping/Mapping.hpp"
+#include "logging/Logger.hpp"
+#include <vector>
+
+namespace precice {
+namespace mapping {
+
+/// Geometric multiscale mapping in axial direction
+class AxialGeoMultiscaleMapping : public Mapping
```

Comment:
I wonder if eventually we should have a `GeoMultiscaleMapping` parent class. Let's just keep this in mind.

Reply:
Would be a good idea if we later on also implement `RadialGeoMultiscaleMapping` as a child.